graylog2-web-interface/src/components/bootstrap/BootstrapModalWrapper.jsx

```diff
@@ -53,6 +65,7 @@ BootstrapModalWrapper.propTypes = {
   ]).isRequired,
   onOpen: PropTypes.func,
   onClose: PropTypes.func,
+  onCancel: PropTypes.func,
```

Comment:
That is correct. I will rename it to `onHide` just as the `Modal` component itself names it.